css/styles.css

```diff
@@ -39,20 +26,22 @@ nav{
     padding-right: 41px;
     padding-top: 26px;
     padding-bottom: 21px;
+    text-decoration: none;
+    color: white;
+    font-family: Myriad Pro;
+    font-size: 24px;
 }
 
 .nav-link:hover{
     cursor: pointer;
     background-color: #222f3a;
-    border-right: 2px solid white;
-    border-left: 2px solid white;
+    outline: 1px solid white;
 }
 
 .aside{
     display: flex;
     flex-direction: column;

```

Comment:
plz try to delete it and nothing will be changed

For this menu my suggestion is to use https://stackblitz.com/edit/js-wnh4co?file=index.js